src/Entity/CommandInterface.php

```diff
@@ -28,4 +28,9 @@ public function getTags(): array;
      * @return bool
      */
     public function hasStopOnError(): bool;
+
+    /**
+     * @return \SprykerSdk\SdkContracts\Entity\ConverterInterface|null
+     */
+    public function getConverter(): ?ConverterInterface;
```

Comment:
Do they have to be nullable?
This makes using them impossible directly, they always need to check for !== null before using.

Reply:
not all commands have convertors

Reply:
Sure, but then consider my recommendations for clean API usage

Reply:
if I add has*() how I implement getConverter(): ConverterInterface; if I don't have this convertor in entity. has*() doesn't make sense here

Reply:
Yeah, orFail() is the correct approach for DTOs
but it can also be added later
so no blocker